packages/react/src/components/button/button.stories.tsx

```diff
@@ -1,89 +1,75 @@
-import { space } from '@onfido/castor';
 import { IconName, iconNames } from '@onfido/castor-icons';
 import { Button, ButtonProps, Icon } from '@onfido/castor-react';
-import React, { SyntheticEvent } from 'react';
-import { Meta, omit, Story, storyOf } from '../../../../../docs';
+import React from 'react';
+import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
 
 export default {
   title: 'React/Button',
   component: Button,
   argTypes: {
-    ...omit<ButtonProps>('className', 'onClick', 'style'),
-    children: { control: 'text' },
+    children: {},
```

Comment:
Do this need to be added, looks like it's just an empty object?

Reply:
All it does is reorder it in the table. We could remove it if you don't mind it being after other props.

Reply:
I believe originally I did put this also for same reason. Well, I think also because it wasn't as "text" by default too.

I think I don't mind it being _after_, still in other stories we already have a mess of argument ordering so TBH no point of fighting...

Reply:
Removed hard-coded ordering, we can look into general story arguments ordering later.